gensim/test/test_ldamodel_distributed.py

```diff
@@ -0,0 +1,527 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
```

Comment:
Is this new source? If yes, then the year should be 2019. If no, then please explain where this is coming from.